### PR TITLE
KOGITO-2653 LoadMore not using businesskeys to retrieve next elements

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

JIRA : https://issues.redhat.com/browse/KOGITO-2653

Description : fixed a bug which failed to retrieve the more data using loadmore when business key filter was applied.

### CODE REVIEW

**Comment by AjayJagan** [comment]
jenkins please retest this

**Comment by nmirasch** [comment]
@AjayJagan  Is there any need to keep two different queries when the only difference is to have that business key filter or not?

**Comment by AjayJagan** [comment]
jenkins please retest this

File: ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx [hunk 1]
```diff
@@ -35,19 +35,19 @@ const LoadMore: React.FC<IOwnProps & OUIAProps> = ({
   };
 
   const load10More = () => {
-    setLoadMoreClicked(true);
+    setLoadMoreClicked && setLoadMoreClicked(true);
```
Comment (nmirasch) [inline ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: @AjayJagan  Is this change related to the reported jira? I see this changeis  in https://github.com/kiegroup/kogito-apps/pull/361 too.
@Sara4994, @AjayJagan we have to avoid include code not related with the linked jira. 
Thanks both to keep improving the code but let's do the things in a way that we can easily track the changes.

Comment (AjayJagan) [reply ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: @nmirasch  - yes I agree.. but without that the loadmore did not work in process list page . So I can remove the changes and wait for @Sara4994 's code to get merged.

**Review by nmirasch — APPROVED** [review]

**Comment by AjayJagan** [comment]
jenkins please retest this

**Review by Sara4994 — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]
